library/src/main/java/com/novoda/downloadmanager/lib/DatabaseHelper.java

```diff
@@ -113,18 +111,13 @@ public void onCreate(@NonNull SQLiteDatabase db) {
      */
     @Override
     public void onUpgrade(@NonNull SQLiteDatabase db, int oldVersion, final int newVersion) {
-        if (oldVersion == 1 && newVersion == 2) {
-            upgradeFromVersionOneToVersionTwo(db);
+        if (oldVersion == OLD_DATABASE_VERSION && newVersion == NEW_DATABASE_VERSION) {
+            upgradeFromVersionTwoToVersionThree(db);
         }
     }
 
-    private void upgradeFromVersionOneToVersionTwo(@NonNull SQLiteDatabase db) {
-        try {
-            db.execSQL(VERSION_ONE_TO_VERSION_TWO_MIGRATION_SCRIPT);
-        } catch (SQLException ex) {
-            LLog.e("couldn't update table in downloads database to v2");
-            throw ex;
-        }
+    private void upgradeFromVersionTwoToVersionThree(@NonNull SQLiteDatabase db) {
+        createBatchesTable(db);
```

Comment:
I thought that I put a comment on this, there is no alter for removing a column, there are three options:
1. Drop
2. Create a temporary table, copy values, drop tmp table
3. Leave this column and is not used